assets/xml/objects/object_wf.xml

```diff
@@ -1,114 +1,128 @@
+
+        <Texture Name="gWolfosNormalEyeHalfTex" OutName="wolfos_normal_eye_half" Format="rgba16" Width="16" Height="16" Offset="0x8368"/>
+        <Texture Name="gWolfosNormalEyeNarrowTex" OutName="wolfos_normal_eye_narrow" Format="rgba16" Width="16" Height="16" Offset="0x8568"/>
+
+        <DList Name="gWolfosNormalTailDL" Offset="0x8F58"/>
+        <DList Name="gWolfosNormalFrontLeftClawDL" Offset="0x8FF8"/>
+        <DList Name="gWolfosNormalFrontRightClawDL" Offset="0x90C0"/>
+        <DList Name="gWolfosNormalHeadDL" Offset="0x9188"/>
+        <DList Name="gWolfosNormalEyesDL" Offset="0x94A8"/>
+
+        <Limb Name="gWolfosNormalRootLimb" LimbType="Standard" Offset="0x9540"/>
+        <Limb Name="gWolfosNormalBackLeftThighLimb" LimbType="Standard" Offset="0x954C"/>
+        <Limb Name="gWolfosNormalBackLeftShinLimb" LimbType="Standard" Offset="0x9558"/>
+        <Limb Name="gWolfosNormalBackLeftPasternLimb" LimbType="Standard" Offset="0x9564"/>
+        <Limb Name="gWolfosNormalBackLeftPawLimb" LimbType="Standard" Offset="0x9570"/>
+        <Limb Name="gWolfosNormalTailLimb" LimbType="Standard" Offset="0x957C"/>
+        <Limb Name="gWolfosNormalAbdomenLimb" LimbType="Standard" Offset="0x9588"/>
+        <Limb Name="gWolfosNormalBackRightThighLimb" LimbType="Standard" Offset="0x9594"/>
+        <Limb Name="gWolfosNormalBackRightShinLimb" LimbType="Standard" Offset="0x95A0"/>
+        <Limb Name="gWolfosNormalBackRightPasternLimb" LimbType="Standard" Offset="0x95AC"/>
+        <Limb Name="gWolfosNormalBackRightPawLimb" LimbType="Standard" Offset="0x95B8"/>
+        <Limb Name="gWolfosNormalThoraxLimb" LimbType="Standard" Offset="0x95C4"/>
+        <Limb Name="gWolfosNormalFrontRightUpperLegLimb" LimbType="Standard" Offset="0x95D0"/>
+        <Limb Name="gWolfosNormalFrontRightLowerLegLimb" LimbType="Standard" Offset="0x95DC"/>
+        <Limb Name="gWolfosNormalFrontRightClawLimb" LimbType="Standard" Offset="0x95E8"/>
+        <Limb Name="gWolfosNormalHeadRootLimb" LimbType="Standard" Offset="0x95F4"/>
+        <Limb Name="gWolfosNormalHeadLimb" LimbType="Standard" Offset="0x9600"/>
+        <Limb Name="gWolfosNormalEyesLimb" LimbType="Standard" Offset="0x960C"/>
+        <Limb Name="gWolfosNormalFrontLeftUpperLegLimb" LimbType="Standard" Offset="0x9618"/>
+        <Limb Name="gWolfosNormalFrontLeftLowerLegLimb" LimbType="Standard" Offset="0x9624"/>
+        <Limb Name="gWolfosNormalFrontLeftClawLimb" LimbType="Standard" Offset="0x9630"/>
+        <Skeleton Name="gWolfosNormalSkel" Type="Flex" LimbType="Standard" Offset="0x9690"/>
+
+        <Animation Name="gWolfosSidesteppingAnim" Offset="0x98C8"/>
         <Blob Name="object_wf_zeroes_Blob_0098D8" Size="0x18" Offset="0x98D8"/>
-        <Animation Name="object_wf_Anim_009B20" Offset="0x9B20"/>
-        <Animation Name="object_wf_Anim_00A4AC" Offset="0xA4AC"/>
+        <Animation Name="gWolfosKnockedbackAnim" Offset="0x9B20"/>
```

Comment:
with how it looks and how it's used I would just name this one `gWolfosDamagedAnim`